Feature/5221 create view em submission popout layout

cw-cvp opened the pull request.

cw-cvp committed:
fix merge conflicts

cw-cvp committed:
fix merge conflicts

cw-cvp committed:
display data in modal

cw-cvp committed:
display data in modal

fanwari1 commented:
@cw-cvp I ran this and I'm not actually seeing any values:
![image](https://github.com/US-EPA-CAMD/easey-ecmps-ui/assets/102816236/76d566e8-1f27-44c7-beed-24a37ae2dd8c)
 Also we made some changes so that certain fields that are in the spec aren't actually needed anymore like Analyst and Representative so those should probably be removed and there are a few others as well so you can look at the table and see what columns we show and only have those in the popout.

Also looking at the wireframe, it looks like there's a title there, plus a close button at the bottom. Also there's a gray background so that users can't click on anything else while the popout is open.

subrataroyCVP commented:
Seeing similar like @fanwari1 . No values in the modal.

cw-cvp committed:
Merge branch 'feature/5221-create-view-em-submission-popout-layout' of https://github.com/US-EPA-CAMD/easey-ecmps-ui into feature/5221-create-view-em-submission-popout-layout

cw-cvp committed:
Merge branch 'develop' into feature/5221-create-view-em-submission-popout-layout

cw-cvp committed:
remove unused modal fields

cw-cvp committed:
add dark background to modal

cw-cvp committed:
Update EmSubmissionData.js

-add openViewEditModalHandler and onRowSelection to getSelect dependencies to fix issue with not using updated data
-wrap openViewEditModalHandler and onRowSelection with useCallback

cw-cvp committed:
add title and cancel button
